modules/web/web-impl/src/main/java/com/enonic/xp/web/impl/exception/ExceptionMapperImpl.java

```diff
@@ -48,6 +48,6 @@ public void throwIfNeeded( final WebResponse res )
 
     private boolean isError( final HttpStatus status )
     {
-        return ( status != null ) && ( status.is4xxClientError() || status.is5xxServerError() );
+        return status != null && status != HttpStatus.IM_A_TEAPOT && ( status.is4xxClientError() || status.is5xxServerError() );
```

Comment:
That is too deep to handle 418 specially. 
BasePortalHandler, WebAppHandler and WebDispatcherServlet use this method (indirectly).
`WebAppHandler` would not enjoy a new behavior.

Please do the special handling of 418 only in Handler that needs it.

Reply:
Moved 418 handling to WebDispatcherServlet